packages/openapi-generator/src/options/options-per-service.ts

```diff
@@ -29,12 +42,73 @@ export interface ServiceOptions {
  */
 export async function getOriginalOptionsPerService(
   configPath: string | undefined
-): Promise<OptionsPerService> {
+): Promise<PartialOptionsPerService> {
   return configPath && existsSync(configPath)
     ? JSON.parse(await readFile(configPath, 'utf8'))
     : {};
 }
 
+/**
+ * Get the options per service for the list of services.
+ * If optionsPerServicePath is not given default values are used for the services.
+ * If optionsPerServicePath is given exisiting values for the services are.
+ * @param inputPaths Paths for the service spec files
+ * @param options Options of the generator
+ * @returns The parsed configuration for all services in relativeServicePaths.
+ */
+export async function getOptionsPerService(
+  inputPaths: AbsoluteAndRelativePath[],
```

Comment:
pass the relative path directly to avoid `.relativePath` everywhere.

Reply:
put it to a variable so it is shorter. If you include the relative path calc in the method it will work with absolute and relative as input which is a bit more resilient I think.